terraform-aws-github-runner/modules/runners/lambdas/runners/src/scale-runners/runners.ts

```diff
@@ -70,27 +70,41 @@ export function resetRunnersCaches() {
   ssmParametersCache.reset();
 }
 
+// Helper function to get the current AWS account ID
+async function getCurrentAccountId(): Promise<string> {
+  const sts = new STS();
+  const identity = await sts.getCallerIdentity().promise();
+  if (!identity.Account) {
+    throw new Error('Unable to retrieve AWS account ID');
+  }
+  return identity.Account;
+}
+
 export async function findAmiID(metrics: Metrics, region: string, filter: string, owners = 'amazon'): Promise<string> {
   const ec2 = new EC2({ region: region });
+  const accountId = await getCurrentAccountId();
+  const allOwners = [accountId, owners];
```

Comment:
These would be a good set of owners to include in the list:

https://github.com/pytorch/ci-infra/blob/c0c1e344db9707d6a70f9f98ec086f49aead1ac5/ali/aws/391835788720/us-east-1/autoscaler-lambda.tf#L76-L81